refactor!: rename & remove multiple options (#377)
brodybits/create-react-native-module: README.md

Options:

  -V, --version                             output the version number
  --package-name <packageName>              The full package name to be used in package.json. Default: react-native-(name in param-case)
  --view                                    Generate the package as a very simple native view component
  --object-class-name                       The name of the object class to be exported by both JavaScript and native code. Default: (name in PascalCase)

>   --native-package-id <nativePackageId>     The Java package identifier used by the Android module (Default: `com.reactlibrary`)

Author: ```suggestion
  --native-package-id <nativePackageId>     [Android] The native Java package identifier used for Android (Default: `com.reactlibrary`)
```

```javascript
{
  name: String, /* The name of the library (mandatory) */
  packageName: String, /* The full package name to be used in package.json. Default: react-native-(name in param-case) */
  view: Boolean, /* Generate the package as a very simple native view component (Default: false) */
  objectClassName: String, /* The name of the object class to be exported by both JavaScript and native code. Default: (name in PascalCase) */
  platforms: Array | String, /* Platforms the library will be created for. (Default: ['android', 'ios']) */

>   nativePackageId: String, /* The Java package identifier used by the Android module (Default: com.reactlibrary) */

Author: ```suggestion
  nativePackageId: String, /* [Android] The native Java package identifier used for Android (Default: `com.reactlibrary`) */
```
